sql/item_mods.sql

```diff
@@ -49721,7 +49721,7 @@ INSERT INTO `item_mods` VALUES (23180,31,36);    -- MEVA: 36
 INSERT INTO `item_mods` VALUES (23180,68,32);    -- EVA: 32
 INSERT INTO `item_mods` VALUES (23180,163,-600); -- DMGMAGIC: -600
 INSERT INTO `item_mods` VALUES (23180,384,400);  -- HASTE_GEAR: 400
-INSERT INTO `item_mods` VALUES (23180,485,1);    -- SHIELD_MASTERY_TP: 1
+INSERT INTO `item_mods` VALUES (23180,485,1);    -- SHIELD_MASTERY_TP: 10
```

Comment:
You didn't change the actual mod value in all 3 entries

Reply:
That would definitely be helpful, fixed.

Reply:
I... can't resolve threads, but cool